test/core/end2end/generate_tests.bzl

```diff
@@ -117,6 +117,7 @@ END2END_FIXTURES = {
     "h2_tls_static_async_tls1_3": _fixture_options(secure = True),
     "h2_tls_certwatch_sync_tls1_2": _fixture_options(secure = True),
     "h2_tls_certwatch_async_tls1_3": _fixture_options(secure = True),
+    "h2_tls_peer_property_external_verifier": _fixture_options(secure = True),
```

Comment:
Do we actually need to add a new test fixture for this?  Adding a test fixture is a pretty big multiplier for the set of test cases, because we basically run every core e2e test case in every fixture.  But it's not clear to me that this particular code is actually useful to run against every one of the e2e test cases -- it seems like we really need to test this functionality only once.

I suggest instead just adding a single one-off test to cover this case with one common config.  As an example of such a one-off e2e test, see:

https://github.com/grpc/grpc/blob/master/test/core/end2end/h2_ssl_session_reuse_test.cc

Reply:
I like the single one-off test better. I wasn't sure if all of the end2end tests for `core` had to be done with the fixture setup. I'll change this out for a one-off

Reply:
Changed, PTAL